views/lib/view.js

```diff
@@ -336,7 +336,8 @@ export default class View {
 
   static getInflectionsData (homonym, options) {
     // Select inflections this view needs
-    return this.dataset.createInflectionSet(this.mainPartOfSpeech, this.getRelatedInflections(homonym.inflections), options)
+    const dataset = this.dataset
+    return dataset.createInflectionSet(this.mainPartOfSpeech, this.getRelatedInflections(homonym.inflections), options)
```

Comment:
What is a difference with the old version? Maybe it's better to have it in one line?

Reply:
I was sure - I fixed it!
Thank you for pointing to this.